fdbserver/CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ set(FDBSERVER_SRCS
   FDBExecHelper.actor.cpp
   FDBExecHelper.actor.h
   fdbserver.actor.cpp
+  GetCipherKeys.actor.cpp
```

Comment:
a nit: would it be better to explicitly name is `GetEncryptionCipherKeys.XX.XX`? For instance, if authorization needs to define an actor for specific usecase, then naming would be important to ensure the file context? 

Again, if you agree we can rename it as a separate PR. 
Thanks